docs/docs/deployment/hosting/kubernetes.md

```diff
@@ -425,6 +425,14 @@ The following table lists the configurable parameters of the chart and their def
 
 ---
 
+## Key upgrade notes
+
+- [0.20.0](https://artifacthub.io/packages/helm/flagsmith/flagsmith/0.20.0): upgrades the bundled in-cluster Postgres.
```

Comment:
I think I'd rather see the release number as a header, then add a section beneath it explaining the changes along with a `WARNING: data loss possible!` or something?